### ISSUE TITLE
Water not working properly 

### ISSUE THREAD

**Issue by MAyaghmay** [Unity-Technologies/BoatAttack#136]
I only use  "com.verasl.water-system" package and add it with package manager.
in editor every thing is working fine but when I build project for android it will show like image on device:
![Screenshot_20210401-073816_tanaruz](https://user-images.githubusercontent.com/7480154/113238315-c8301280-92bd-11eb-8b5e-8c151bc85fd2.jpg)

for just a sec water shows properly but after that it goes like that. 
I just set all the setting as in boatAttack project (player setting, renderer, etc... )

**Comment by antont** [comment]
Hi - am seeing the same, did you get to figure this out?

**Comment by antont** [comment]
I actually have it working when built from the original project, and I think now also when using the assets in my own project .. had to copy the cam setup over too I think, still testing and investigating how it renders exactly

**Comment by antont** [comment]
Yep I did get it working! Last problem was that the shader or something stopped working after a sec, but that was with GLES - read comments here earlier and removed Vulkan and tried with GLES 3.2 earlier, but disabled GL now and use Vulkan instead and it works.

**Comment by MAyaghmay** [comment]
> Yep I did get it working! Last problem was that the shader or something stopped working after a sec, but that was with GLES - read comments here earlier and removed Vulkan and tried with GLES 3.2 earlier, but disabled GL now and use Vulkan instead and it works.

Hi, I disabled the Vulkan and only use GLES will try you approach as well. thanks a lot.

**Comment by iamyawer** [comment]
Yeah I see the same issue with water, @antont with vulkan graphics api do you know where to get target device that supports vulkan api. Coz with openGL ES version same issue arises.

**Comment by MAyaghmay** [comment]
> Yeah I see the same issue with water, @antont with vulkan graphics api do you know where to get target device that supports vulkan api. Coz with openGL ES version same issue arises.

I test it with vulkan but no luck. couldn't make it to work properly.

**Comment by shubham-rgupta** [comment]
![image](https://github.com/Unity-Technologies/BoatAttack/assets/71450289/2c4592f3-0df8-4d73-8a47-a7fbd215267e)
seeing this in editor and in android build also
